debug/org.eclipse.cdt.debug.ui/src/org/eclipse/cdt/debug/internal/ui/commands/Messages.properties

```diff
@@ -14,3 +14,4 @@ ReverseDebugging_UndefinedTraceMethod=Undefined trace method for Reverse Debuggi
 ReverseDebugging_ToggleHardwareTrace=Toggle Hardware Trace
 ReverseDebugging_ToggleSoftwareTrace=Toggle Software Trace
 ReverseDebugging_ToggleReverseDebugging=Toggle Reverse Debugging
+ToogleCommand_State_Not_found=Toggle state not found for command: {0}
```

Comment:
```suggestion
# {0} - command id
ToogleCommand_State_Not_found=Toggle state not found for command: {0}
```

Please add the comment to make it easier for translators to know what the parameter contains.

Reply:
Fixed.